pandas/core/frame.py

```diff
@@ -6990,8 +6982,9 @@ def applymap(self, func) -> "DataFrame":
         0  3  4
         1  5  5
 
-        Note that a vectorized version of `func` often exists, which will
-        be much faster. You could square each number elementwise.
+        Note that a vectorized version of `func` often exists,
+        which will be much faster.
+        You could square each number elementwise.
```

Comment:
can you revert this

Reply:
Sorry, for being late! I'm having a very busy semester. I'll do this asap and get back!